js/client-teambuilder.js

```diff
@@ -2851,6 +2851,8 @@
 						this.unChooseMove(curVal);
 						$inputEl.val('');
 						delete this.search.cur[toID(val)];
+						// Remove all instances of the duplicate move from the moves array.
+						this.curSet.moves = this.curSet.moves.filter(move => move != curVal);
```

Comment:
Hm, this works but uses modern JavaScript that would break older browsers. The CI should have caught it; I'm not sure why the CI isn't running on your code.